tests/integration/test_fim/test_synchronization/test_sync_overlap.py

```diff
@@ -0,0 +1,180 @@
+    '''
+    description: Check if the 'wazuh-syscheckd' daemon is performing a synchronization at the interval specified in the
+                 configuration, using the 'interval' tag, if a new synchronization is fired, and the last sync message
+                 has been recieved in less time than 'response_timeout, the sync interval is doubled.
+                 The new value for interval cannot be higher than max_interval option. After a new sync interval is
+                 tried and the last message was recieved in a time that is higher than response_timeout, the sync
+                 interval value is returned to the configured value.
+
+    test_phases:
+        - Create a folder with a number of files inside.
+        - Restart syscheckd.
+        - Check that a sync interval started.
+        - Check that next sync is skipped and interval value is doubled
+        - Check that interval value is returned to configured value after successful sync
+
+    wazuh_min_version: 4.5.0
+
+    tier: 2
+
+    parameters:
+        - configuration:
+            type: dict
+            brief: Configuration values for ossec.conf.
+        - metadata:
+            type: dict
+            brief: Test case data.
+        - set_wazuh_configuration_fim:
+            type: fixture
+            brief: Set ossec.conf and local_internal_options configuration.
+        - create_files_in_folder:
+            type: fixture
+            brief: create files in monitored folder, and delete them after the test.
+        - restart_syscheck_function:
+            type: fixture
+            brief: restart syscheckd daemon, and truncate the ossec.log.
+        - wait_for_fim_start_function:
+            type: fixture
+            brief: check that the starting fim scan is detected.
+
+    assertions:
+        - Verify that the new value for interval when doubled is equal or lower to max_interval.
+
+    input_description: A test case (sync_interval) is contained in external YAML file (cases_sync_overlap.yaml) which
+                       includes configuration settings for the 'wazuh-syscheckd' daemon. That is combined with
+                       the interval periods and the testing directory to be monitored defined in this module.
+
+    expected_output:
+        - r'Initializing FIM Integrity Synchronization check'
+        - r"*Sync still in progress. Skipped next sync and increased interval.*'(\\d+)s'"
+        - r".*Previous sync was successful. Sync interval is reset to: '(\\d+)s'"
+
+    tags:
+        - scheduled
+    '''
+
+    wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+    # Wait for new scan and check files have been created
+    wazuh_log_monitor.start(timeout=global_parameters.default_timeout, callback=callback_detect_synchronization,
+                            error_message='Did not receive expected '
+                                          '"Initializing FIM Integrity Synchronization check" event',
```

Comment:
You use text here, but you should add a variable in the error section on the `deps/wazuh_testing/wazuh_testing/modules/fim/__init__.py` file and use this variable defined here.